tests/copylat.cpp

```diff
@@ -49,10 +51,11 @@ int main(int argc, char *argv[])
     bool do_cumemcpy = false;
     struct timespec beg, end;
     double lat_us;
+    double bw;
```

Comment:
Isn’t this redundant with copybw?

If you want to do shmoo for bw, is it better to rename the test? “copylat” doesn’t sound right anymore in that case.